OpenMTR/Utilities/CannyFilter.cs

```diff
@@ -11,7 +11,7 @@ public static class CannyFilter
     {
         public static void ApplyCannyFilter(Mat sourceImage, Mat destinationImage)
         {
-            Cv2.Canny(sourceImage, destinationImage, 100, 150);
+            Cv2.Canny(sourceImage, destinationImage, 100, 200);
```

Comment:
We'll need to have a discussion on if the methods wrapping filters are necessary. I initially thought this might a good idea to help learn the library, but as we delve deeper I am seeing the need to utilize filters with different settings depending on certain scenarios.

Reply:
Yeah, I agree. Sometimes readability is nice when wrapping certain methods, but at the same time, it's more code. Definitely if we do decide to keep wrapping these, we need to make it where it can have values passed in for these parameters.